components/org.wso2.identity.webhook.common.event.handler/src/main/java/org/wso2/identity/webhook/common/event/handler/EventConfigManager.java

```diff
@@ -0,0 +1,129 @@
+
+        try {
+            ResourceConfig eventConfigObject = getEventConfig(eventName);
+            if (eventConfigObject.getConfigs() != null &&
+                    eventConfigObject.getConfigs().containsKey(Constants.EVENT_CONFIG_SCHEMA_NAME_KEY)) {
+                return (String) eventConfigObject.getConfigs().get(Constants.EVENT_CONFIG_SCHEMA_NAME_KEY);
+            } else {
+                throw new IdentityEventServerException("Event schema not found in the resource event config " +
+                        "for the eventName: " + eventName);
+            }
+        } catch (ClassCastException e) {
+            throw new IdentityEventServerException("Error while casting event config at server side", e);
+        }
+    }
+
+    /**
+     * Retrieve the event config.
+     *
+     * @param eventName Event name.
+     * @return Resource config object.
+     * @throws IdentityEventServerException If an error occurs.
+     */
+    private ResourceConfig getEventConfig(String eventName) throws IdentityEventServerException {
+
+        JSONObject eventsConfigObject = (JSONObject) getEventsSchemaResourceFile().getConfigs()
+                .get(Constants.EVENT_SCHEMA_EVENTS_KEY);
+        if (eventsConfigObject != null && !eventsConfigObject.isEmpty() &&
+                eventsConfigObject.containsKey(eventName)) {
+            return new ResourceConfig((JSONObject) eventsConfigObject.get(eventName));
+        } else {
+            throw new IdentityEventServerException("Event schema not found in the resource event config " +
+                    "for the eventName: " + eventName);
+        }
+    }
+
+    /**
+     * This method reads the event schema resource file and returns the config object.
+     *
+     * @return Config object with content in the resource file.
+     * @throws IdentityEventServerException If an error occurs while reading the resource file.
+     */
+    private ResourceConfig getEventsSchemaResourceFile() throws IdentityEventServerException {
+
+        if (eventSchema == null) {
+            synchronized (lock) {
+                if (eventSchema == null) {
+                    String resourceFilePath = new File(".").getAbsolutePath() + File.separator +
+                            Constants.EVENT_PUBLISHER_EVENT_SCHEMA_RESOURCE_FILE_PATH;
+                    JSONParser jsonParser = new JSONParser();
+                    try {
+                        JSONObject eventConfigJSON = (JSONObject) jsonParser.parse(new InputStreamReader(
+                                Files.newInputStream(Paths.get(resourceFilePath)), StandardCharsets.UTF_8)
+                        );
+                        eventSchema = new ResourceConfig(eventConfigJSON);
+                    } catch (IOException | ParseException | ClassCastException e) {
+                        throw new IdentityEventServerException("Error while reading the event schema file", e);
```

Comment:
Better specify the file name so it gives more context

Reply:
Addressed with https://github.com/wso2-extensions/identity-webhook-event-handlers/pull/1/commits/dc72213e0cb8089d982fddd20de1d085f01ac28a